src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenDeconstructTests.cs

```diff
@@ -480,7 +480,7 @@ public void Deconstruct(out int a, out string b, out params int[] c)
 
             var comp = CreateStandardCompilation(source);
             comp.VerifyDiagnostics(
-                // (4,58): error CS8404:  The parameter modifier 'params' cannot be used with 'out' 
+                // (4,58): error CS8404:  The parameter modifier 'params' cannot be used after the modifier 'out' 
```

Comment:
I think this is irrelevant. The message doesn't mean the opposite is true.
This changed because parameters now have to appear in certain orderings.

---
In reply to: [120727103](https://github.com/dotnet/roslyn/pull/19703#discussion_r120727103) [](ancestors = 120727103)